lab3/lab3_task_2.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>lab3_2</title>
+    <style>
+        a[href^="http"] {
+            color: #f70            
+        }        
+        a[href*="google"] {
+            color: #666
+        }
+        a[target="_blank"]:after {                        
+            content: "⇱\00a0";            
+            display: inline-block;
+            text-decoration-line: none;
+            transform: rotateY(180deg);        
+        }
+    </style>
+</head>
+<body>
+
+<div>
+    <ul>
+        <li><a href="/">Главная</a></li>
+        <li><a href="/news">Новости</a></li>
+        <li><a href="http://ya.ru">Яндекс</a></li>
```

Comment:
In such cases better always add the last slash right away, like `href="http://ya.ru/"`. Otherwise you will trigger an HTTP redirect every time, resulting in two HTTP requests. Check it yourself in the Networking tab of Chrome Dev Tools.

Reply:
cool, I didn't know that. thanks!